net.sourceforge.squirrel_sql.desktop

```diff
@@ -0,0 +1,5 @@
+[Desktop Entry]
```

Comment:
Ideally should be upstreamed.

Reply:
I put in a merge request:
https://sourceforge.net/p/squirrel-sql/git/merge-requests/75/